students/eunkyung/users/views.py

```diff
@@ -25,10 +25,12 @@ def post(self, request):
             if User.objects.filter(email=email).exists():
                 return JsonResponse({'MESSAGE':'ALREADY_EXISTS_EMAIL'}, status=409)
 
+            hashed_password = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt())
+
             User.objects.create(
                name     = name,
                email    = email,
-               password = password,
+               password = hashed_password.decode('utf-8'),
```

Comment:
디코딩을 위에서 hashed_password 변수 설정할 때 미리 해주면 좋을 것 같습니다!
```suggestion
               password = hashed_password,
```